Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/HttpTriggerCSharp.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Extensions.Http;
+using Microsoft.Azure.Functions.Worker.Pipeline;
+using Microsoft.Extensions.Logging;
+
+namespace Company.Function
+{
+    public static class HttpTriggerCSharp
+    {
+        [FunctionName("HttpTriggerCSharp")]
+        public static HttpResponseData Run([HttpTrigger(AuthorizationLevel.AuthLevelValue, "get", "post")] HttpRequestData req,
+            FunctionExecutionContext executionContext)
+        {
+            var logger = executionContext.Logger;
+            logger.LogInformation("C# HTTP trigger function processed a request.");
+
+            var response = new HttpResponseData(HttpStatusCode.OK);
+            var headers = new Dictionary<string, string>();
+            headers.Add("Date", "Mon, 18 Jul 2016 16:06:00 GMT");
```

Comment:
Do we really need to hard-code a date like this? Seems a bit odd for the default template

Reply:
This entire template code will actually change in a week or so. I just put what we had in samples/ as placeholder. But we are actively updating that. As soon as we release Nugets and updated samples this week, I will update the contents here.